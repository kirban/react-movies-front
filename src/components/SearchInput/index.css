.searchInput {
    width: 45rem;
    height: 4rem;
    margin-right: 1rem;

    font-family: Montserrat;
    font-size: 20px;
    font-style: normal;
    font-weight: 400;
    line-height: 24px;
    letter-spacing: 0px;
    text-align: left;

    background: rgba(50, 50, 50, 0.8);
    mix-blend-mode: normal;
    opacity: 0.7;
    border-radius: 4px;
}

.searchButton {
    width: 15rem;
    height: 4rem;

    font-family: Montserrat;
    font-size: 20px;
    font-style: normal;
    font-weight: 500;
    line-height: 24px;
    letter-spacing: 0px;
    text-align: center;
    text-transform: uppercase;
    color: #FFF;

    background: #F65261;
    border-radius: 4px;
}