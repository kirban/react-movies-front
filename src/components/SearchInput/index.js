import React from 'react';
import './index.css';

const SearchInput = () => {
  return (
    <form action="search.php">
      <input className="searchInput" type="text" name="search" id="" placeholder="What do you want to watch?" />
      <input className="searchButton" type="submit" value="Search" />
    </form>
  );
}

export default SearchInput
