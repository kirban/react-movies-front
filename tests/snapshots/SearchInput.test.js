import React from 'react'
import { render, cleanup } from '@testing-library/react'
import SearchInput from '@components/SearchInput/index'

jest.mock('react-router', () => ({
    ...jest.requireActual('react-router'),
    useParams: jest.fn().mockReturnValue({ query: 'test' })
}))

afterEach(cleanup)

it("renders search panel", () => {
    const { asFragment } = render(<SearchInput />)
    expect(asFragment(<SearchInput />)).toMatchSnapshot()
})