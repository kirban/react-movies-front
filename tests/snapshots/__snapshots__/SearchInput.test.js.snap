// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders search panel 1`] = `
<DocumentFragment>
  <form>
    <input
      class="searchInput"
      id=""
      name="search"
      placeholder="What do you want to watch?"
      type="text"
      value="test"
    />
    <input
      class="searchButton"
      type="submit"
      value="Search"
    />
  </form>
</DocumentFragment>
`;
